### PR TITLE
Cypress/E2E: Promote tests to prod

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Promote tests to prod

### CODE REVIEW

**Review by josephbaylon — APPROVED** [review]
LGTM

File: e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js [hunk 1]
```diff
@@ -7,6 +7,7 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
+// Stage: @prod
```
Comment (prapti) [inline e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js:10]: T1672_1 from this spec has failed in all 5 latest runs. T1678 has been failing occasionally as well.

**Review by prapti — CHANGES_REQUESTED** [review]
Hi Saturn, most of the promotions look good, however, looks like at least 2 tests from `archived_leave_channel_spec` are failing quite consistently in the last 4 runs (please see the comment below). We should probably leave this one spec file out and promote the rest.

File: e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js:10]: Oh, right. I thought this https://github.com/mattermost/mattermost-webapp/pull/7765 was already merged.

Comment (saturninoabril) [reply e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js]: Now removed, PTAL, thanks Prapti!

**Review by prapti — APPROVED** [review]
LGTM!
Thanks Saturn!

**Review by furqanmlk — APPROVED** [review]
image
image
image
image
image
image
LGTM

**Pull request merged**